src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -5,42 +5,52 @@
 import edu.uc.inventorymanager.dto.ItemStatus;
 import edu.uc.inventorymanager.dto.User;
 import edu.uc.inventorymanager.service.IItemService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
 
+
 @Controller
 public class MainController {
+
+    static final String USER_NAME = "John Doe";
+    static final String DESCRIPTION = "Des";
+
     @Autowired
     IItemService itemService;
 
+    Logger log = LoggerFactory.getLogger(this.getClass());
+
```

Comment:
Added constants to commonly used variables.
It looks like some hardcoded placeholders such as John Doe are being used in this code. For that reason, I think it’s a good idea to put them into a constant variable, so they can be changed easily if there ever be a need. This also makes understanding the code easier. 
Helpful link - https://www.javatpoint.com/java-constant